samples/testing-frameworks/appium/server-side/java/src/test/java/AbstractAppiumTest.java

```diff
@@ -56,32 +56,16 @@ public static AppiumDriver getIOSDriver() throws Exception {
         if (platformVersion == null) {
             platformVersion = "";
         }
-        if (automationName == null) {
-            DefaultArtifactVersion version = new DefaultArtifactVersion(ideviceinfoCheck("ProductVersion"));
-            DefaultArtifactVersion minVersion = new DefaultArtifactVersion("9.3.5");
-            // Use XCUITest if device is above iOS version 9.3.5
-            if (version.compareTo(minVersion) >= 0) {
-                automationName = "XCUITest";
-            } else {
-                automationName = "Appium";
-            }
-        }
-        if (udid == null) {
-            udid = ideviceinfoCheck("UniqueDeviceID");
-        }
 
         DesiredCapabilities capabilities = new DesiredCapabilities();
-        capabilities.setCapability("automationName", automationName);
+        DesiredCapabilities appiumOptions = new DesiredCapabilities();
+        appiumOptions.setCapability("automationName", "XCUITest");
         capabilities.setCapability("platformName", "IOS");
-        capabilities.setCapability("deviceName", deviceName);
         if (udid != null) {
-            capabilities.setCapability("udid", udid);
+            appiumOptions.setCapability("udid", udid);
         }
-        if (platformVersion != null) {
-            capabilities.setCapability("platformVersion", platformVersion);
-        }
-        capabilities.setCapability("app", appFile);
-        capabilities.setCapability("newCommandTimeout", 120);
+        appiumOptions.setCapability("app", appFile);
+        capabilities.setCapability("appium:options", appiumOptions);
```

Comment:
Please sort order of lines so appiumOptions and capabilities go together.

Reply:
Done